skopeo/README.md

```diff
@@ -0,0 +1,102 @@
+# Skopeo
+
+
+[Skopeo](https://github.com/containers/skopeo) is a command line tool for working with remote image registries. Skopeo doesn’t require a daemon to be running while performing its operations. In particular, the handy skopeo command called `copy` will ease the whole image copy operation. Without further ado, you can copy an image from a registry to another simply by running:
+```
+skopeo copy docker://internal.registry/myimage:latest /
+docker://production.registry/myimage:v1.0
+```
+The copy command will take care of copying the image from `internal.registry` to `production.registry`
+
+If your production registry requires credentials to login in order to push the image, skopeo can handle that as well.
+
+```
+skopeo copy --dest-creds prod_user:prod_pass docker://internal.registry/myimage:latest /
+docker://production.registry/myimage:v1.0
+```
+
+The same goes for credentials for the source registry (internal.registry) by using the `--src-creds` flag.
+
+It is also useful for copying images between two remote docker registries, such as the registries of two different OpenShift clusters, as shown
+```
+skopeo copy docker://busybox:latest oci:busybox_ocilayout:latest
+```
+Skopeo copy isn’t limited to remote containers registries. The image prefix `docker://` from the above command define the transport to be used when handling the image.
+
+There are others also similar to that:
+
+- atomic
+- containers-storage
+- dir
+- docker
+- docker-daemon
+- docker-tar
+- oci
+- ostree
+
+This `task` can be used to copy one or more than one images to-and fro various storage mechanisms.
+
+This task uses image from [this](https://github.com/tektoncd/plumbing/tree/master/tekton/images/skopeo) Dockerfile.
+
+## Install the Task
+
+```
+kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/v1beta1/skopeo/skopeo.yaml
+```
+
+## Parameters
+
+- **srcImageURL**: The URL of the image to be copied to the `destination` registry.
+- **destImageURL**: The URL of the image where the image from `source` should be copied to.
+
+## Resources
+
+* `Secret` to provide the credentials of the source and destination registry where the image needs to be copied from and to.
+* `ConfigMap` to provide support for copying multiple images, this contains file `url.txt` which stores images registry URL's.
+* `Workspace` to mount file containing multiple source and destination images registries URL, which is mounted as configMap.
+
+[This](../skopeo/example/quay-secret.yaml) example can help to use secrets for providing credentials.
```

Comment:
I don't think this should be `Resources` as usually we refer to `Resources` for the PipelineResource. 
You should have a `## Workspace` heading and one for configmap and secret most likely

Reply:
Okay, will update this. Thanks for the review :)